components/places/ios/Places/Bookmark.swift

```diff
@@ -24,3 +24,246 @@ public enum BookmarkRoots {
         BookmarkRoots.UnfiledFolderGUID,
     ])
 }
+
+// Keeping `BookmarkNodeType` in the swift wrapper because the iOS code relies on the raw value of the variants of
```

Comment:
See [here](https://github.com/mozilla-mobile/firefox-ios/blob/0dab728a700df53fe5b9883d697a13ad59e1252e/Storage/SQL/BrowserSchema.swift#L623) for details.

Reply:
Is that sql actually used? I don't quite understand how that sql is executed against our database?

Reply:
Oh, good point. I trusted the comment above the view without really asking why it existed, along with the tables backing it. Anyway, I've discussed this with Nish and he's similarly unsure of the necessity of that code so I've created [this ticket](https://github.com/mozilla-mobile/firefox-ios/issues/9760) to make sure this is addressed.